src/문자열 압축.js

```diff
@@ -0,0 +1,54 @@
+function solution(s) {
+  let answer = 0;
+  let halfSizeOfS = Math.floor(s.length / 2);
+
+  if (s.length === 1) return 1;
+
+  for (let i = 1; i <= halfSizeOfS; i++) {
+    const result = compressArray(spliter(s, i)).length;
+
+    if (!answer) {
+      answer = result;
+    }
+
+    if (result < answer) {
+      answer = result;
+    }
+  }
+
+  return answer;
```

Comment:
```suggestion
     answer = Math.min(answer, result);
  }

  return answer;
```

이렇게 하면 조금 더 직관적으로 코드를 짤 수 있을꺼 같습니다!

Reply:
요렇게 하려면 answer의 초기값을  Infinity로 해야겠군요. 조언 감사합니다 ㅎ